datajoint/fetch.py

```diff
@@ -20,46 +25,57 @@ def prepare_attributes(relation, item):
         raise DataJointError("Index must be a slice, a tuple, a list, a string.")
     return item, attributes
 
-class FetchQuery:
+def copy_first(f):
+    @wraps(f)
+    def ret(*args, **kwargs):
+        args = list(args)
+        args[0] = args[0].__class__(args[0]) # call copy constructor
+        return f(*args, **kwargs)
 
-    def __init__(self, relation):
-        """
+    return ret
 
-        """
-        self.behavior = dict(
-            offset=0, limit=None, order_by=None, descending=False, as_dict=False, map=None
-        )
-        self._relation = relation
+class Fetch:
+    def __init__(self, relation):
+        if isinstance(relation, Fetch): # copy constructor
+            self.behavior = dict(relation.behavior)
+            self._relation = relation._relation
+        else:
+            self.behavior = dict(
+                offset=0, limit=None, order_by=None, as_dict=False
+            )
+            self._relation = relation
 
 
+    @copy_first
     def from_to(self, fro, to):
         self.behavior['offset'] = fro
```

Comment:
Either `order`, `limit`, and `offset` or `order_by`, `limit_to`, `offset_by`. Since we are not MySQL there is no need to adopt inconsistent naming conventions. I like the second option better because statements read clearer (like `populated_from`). We could make a poll among existing datajoint-python users which statement they prefer.